proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -22,7 +22,7 @@ message SimpleMetricValue {
 }
 
 // Encapsulates derived statistical summaries of a single metric.
-//
+c//
```

Comment:
Whoops!

Reply:
This commit was added by mistake and has now been removed.